index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personnal Portfolio Web</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Work</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+     </nav>
+     <section id="welcome-section">
+         <h1>Hey I am Mimic</h1>
+         <p>A web developer</p>
+     </section>
+     <section id="projects">
+         <h2>
+             These are some of my projects
+         </h2>
+         <hr>
+         <div class="project-tile">
+             <a id="profile-link" href="#" target="_blank">
+                <img src="https://images.unsplash.com/photo-1488590528505-98d2b5aba04b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2670&q=80" alt="Tech">
+                <h4>WebApp xxx</h4>
+             </a>
+             <a id="profile-link" href="#" target="_blank">
+                <img src="https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2670&q=80" alt="Tech">
+                <h4>Pair Programming</h4>
+             </a>
+             <a id="profile-link" href="#" target="_blank">
+                <img src="https://images.unsplash.com/photo-1664575262619-b28fef7a40a4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1132&q=80" alt="Tech">
+                <h4>Work Remotely</h4>
+             </a>
+         </div>
+         <button id="show-all-button">Show All</button>
+     </section>
+     <section id="social-section">
+        <h2>Let's work together...</h2>
+        <p>How do you take your coffee?</p>
+        <ul>
+            <li><a href="#">Facebook</a></li>
+            <li><a href="#">GitHub</a></li>
+            <li><a href="#">Email</a></li>
+            <li><a href="#">LinkedIn</a></li>
+        </ul>
+     </section>
+     <hr id="hr-before-footer">
+     <section id="footer-section">
+        <h3>Personnal Portfloio Website Microverse Trial</h3>
+     </section>
```

Comment:
Please use the footer tag over the section tag. The footer HTML element represents a footer to its nearest content. It has information such as author, copyright data and information related to the document or webpage.

Reply:
[Optional] You did good with the review here, however, I strongly suggest you use back strings to highlight tags. e.g `footer`